tidy3d/plugins/adjoint/utils/penalty.py

```diff
@@ -70,6 +79,17 @@ def evaluate(self, points: ArrayFloat2D) -> float:
         desired minimum value. If ``wrap``, it is assumed that the points wrap around to form a
         closed geometry instead of an isolated line segment."""
 
+        if not is_jax_object(points):
+            log.warning(
+                "The points passed to 'RadiusPenalty.evaluate()' are not a 'jax' array. "
+                "If you are passing the 'JaxPolySlab.vertices' field directly, note that the "
+                "derivative information for this field is no longer traced by as of 'tidy3d 2.7'. "
```

Comment:
- I always find using "you" in these messages a bit strange. Here it can be easily changed, `If you are passing` -> `If passing`. In other places it may be more awkward to rephrase, so not sure about a general policy. But I feel like usually packages are not talking to "you", i.e. I feel like there is some implicit standard about this.

- ~~by~~ as of

- `'tidy3d 2.7'` seems a bit strange formatting? Do we put `''` around package names (also `'jax'` here)? And even if so maybe `'tidy3d' v2.7`?

Reply:
changed to 
```
"The points passed to 'RadiusPenalty.evaluate()' are not a 'jax' array. If passing the 'JaxPolySlab.vertices' field directly, note that the derivative information for this field is no longer traced by jax as of version '2.7'. The derivative information is contained in 'JaxPolySlab.vertices_jax'.  Therefore, we recommend changing your code to either pass that field or pass the output of the parameterization functions directly, eg. 'penalty.evaluate(make_vertices(params))'."
```